cabal-install/tests/IntegrationTests2.hs

```diff
@@ -0,0 +1,258 @@
+module Main where
+
+import Distribution.Client.DistDirLayout
```

Comment:
This will slow down build times on Travis. Would be nice if we could use an internal library in `cabal-install.cabal`.